stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/internal/MessageInputFieldView.kt

```diff
@@ -235,7 +235,8 @@ internal class MessageInputFieldView : FrameLayout {
         switchToMessageMode()
         binding.messageReplyView.setMessage(
             currentMode.repliedMessage,
-            ChatDomain.instance().currentUser.id == currentMode.repliedMessage.user.id
+            ChatDomain.instance().currentUser.id == currentMode.repliedMessage.user.id,
+            null,
```

Comment:
`MessageInputView` should have a separate style, right?

Reply:
Yes